ha/k8s_setup/ha_start.py

```diff
@@ -13,7 +15,8 @@
                                     )
 
 my_parser.add_argument('--services', '-s', help='service_name', required=True, action='store')
-my_parser.add_argument('--config', '-c', help='config file', required=False, action='store')
+my_parser.add_argument('--config', '-c', help='config file url', required=False, action='store')
+my_parser.add_argument('--debug', '-d', help='debug level {0|1}', type=int, nargs='?', const=1, default=0, required=False, action='store')
```

Comment:
This and the remaining code should not be in mainstream.  You can have it under some option (that ensures that this is created only for dev build) or add TODO here for the same and file a ticket for that TODO

Reply:
done